draft-ietf-tls-exported-authenticator.md

```diff
@@ -35,44 +35,52 @@ informative:
 
 This document describes a mechanism in Transport Layer Security (TLS) to
 provide an exportable proof of ownership of a certificate that can be
-transmitted out of band and verified by the other party.
+transmitted out of band and verified by the peer.
 
 --- middle
 
 # Introduction
 
 This document provides a way to authenticate one party of a Transport Layer
-Security (TLS) communication to another using a certificate after the session
+Security (TLS) communication to its peer using a certificate after the session
 has been established.  This allows both the client and server to prove ownership
 of additional identities at any time after the handshake has completed.  This
 proof of authentication can be exported and transmitted out of band from one
-party to be validated by the other party.
+party to be validated by its peer.
 
 This mechanism provides two advantages over the authentication that TLS natively
 provides:
 
 multiple identities -
 
 : Endpoints that are authoritative for multiple identities - but do not have a
-  single certificate that includes all of the identities - can authenticate with
-  those identities over a single connection.
+  single certificate that includes all of the identities - can authenticate additional
+  identities over a single connection.
 
 spontaneous authentication -
 
 : Endpoints can authenticate after a connection is established, in response to
   events in a higher-layer protocol, as well as integrating more context.
 
-This document intends to replace much of the functionality of renegotiation
-in previous versions of TLS.  It has the advantages over renegotiation of not
-requiring additional on-the-wire changes during a connection.  For simplicity,
-only TLS 1.2 and later are supported.
+Versions of TLS prior to TLS 1.3 used renegotiation as a way to enable
+post-handshake client authentication given an existing TLS connection.
+The mechanism described in this document may be used to replace the
+post-handshake authentication functionality provided by renegotiation.
+Unlike renegotiation, exported Authenticator-based post-handshake
+authentication does not require any on-the-wire changes.
```

Comment:
We probably want to qualify the lack of changes to "at the TLS level" or "to the cryptographic keys used" or something like that, as there is clearly some change in the application's bits on the wire to use EAs.